.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "all",
+  "singleQuote": true,
+  "arrowParens": "always"
```

Comment:
Is `"arrowParens": "always"` why you write `(rule) => ...` rather than `rule => ...` ? I actually prefer the latter.

Reply:
Yeah, it's an Airbnb configuration, but we can turn it off